src/util/index.ts

```diff
@@ -1152,7 +1152,10 @@ export function maybeLoadFromExternalFile(filePath: string | object | Function |
     return filePath;
   }
 
-  const finalPath = path.resolve(cliState.basePath || '', filePath.slice('file://'.length));
+  // Render the file path using Nunjucks
+  const renderedFilePath = nunjucks.renderString(filePath, { env: process.env });
```

Comment:
We already add this here: https://github.com/promptfoo/promptfoo/blob/main/src/util/templates.ts#L30 

```suggestion
  const renderedFilePath = nunjucks.renderString(filePath);
```

Reply:
nvm. Decided to use global nunjucks engine instead https://github.com/promptfoo/promptfoo/pull/1688/commits/f94359e10cfad8e330e2bd9a25121811fa0efb0e

Reply:
@mldangelo Decided to use global nunjucks engine instead to centralize the env loading

```javascript
const renderedFilePath = getNunjucksEngine().renderString(filePath, {});
```